source/frontend/src/hooks/repositories/mapLayer/useParcelMapLayer.ts

```diff
@@ -24,61 +25,85 @@ export const useParcelMapLayer = () => {
 
   const { execute: getAllFeatures, loading: getAllFeaturesLoading } = getAllFeaturesWrapper;
 
+  const handleError = useCallback((error: unknown) => {
+    toast.error('Unable to contact Parcel Map');
+  }, []);
```

Comment:
at some point in the future we should improve the network slice to maintain the state of the requests and use a component that monitors it to show or not show errors. For now this is just using the regular toasts as the other services